smart_fetch/cli/main.py

```diff
@@ -21,6 +21,7 @@ def define_parser() -> argparse.ArgumentParser:
         subparsers.add_parser("crawl", help="run a crawl (REST alternative to bulk export)")
     )
     hydrate.make_subparser(subparsers.add_parser("hydrate", help="add to already-exported data"))
+    single.make_subparser(subparsers.add_parser("single", help="request a single resource"))
```

Comment:
Let's chat about the name `single` -- I also toyed with `query`, `direct`, `get` or something along those lines. But ended up with `single` to really emphasize this command against the other bulk-access commands. But I am inviting other votes here.

Reply:
i like this in this context